tests/unit/phpDocumentor/Renderer/ActionHandlerLocatorTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * This file is part of phpDocumentor.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @copyright 2010-2015 Mike van Riel<mike@phpdoc.org>
+ * @license   http://www.opensource.org/licenses/mit-license.php MIT
+ * @link      http://phpdoc.org
+ */
+
+namespace phpDocumentor\Renderer;
+
+use Mockery as m;
+
+/**
+ * Tests the functionality for the ActionHandlerLocator class.
+ */
+class ActionHandlerLocatorTest extends \PHPUnit_Framework_TestCase
+{
+    public function testIfLocatorRetrievesAnActionWithHanderAppendedToClassName()
+    {
+        $action = m::mock('phpDocumentor\Renderer\Action');
```

Comment:
I would use the ::class constant here, Helps when you refactor the name of a class